nuklear.h

```diff
@@ -19798,7 +19798,10 @@ nk_panel_end(struct nk_context *ctx)
         {
             /* horizontal scrollbar */
             nk_flags state = 0;
-            scroll.x = layout->bounds.x;
+            if(layout->flags & NK_WINDOW_SCALE_LEFT)
```

Comment:
``` suggestion
           /* The horizontal scrollbar has priority and can't scale otherwise. Reposition it to avoid overlap with the scaler. */
           if(layout->flags & NK_WINDOW_SCALE_LEFT)
```